### DISCUSSION TITLE
Change from 24 to 12 hours in Bip U Pro model

### DISCUSSION THREAD

**Discussion by ricardfs** [zepp-health/zeppos-forum#118] [Q&A]
Hello partners.

I am creating a Watch Face for the Bip U Pro model but I have a small problem with the hours element, I want to make the hours display in 12-hour format, that is, AM and PM, the fact is that I cannot find how change the format from 24 to 12 hours.

Can you help me please?

Thank you so much.

**Comment by Xuemuyang** [comment]
Hello, you can refer to this API [getTimeFormat](https://docs.zepp.com/docs/1.0/watchface/api/hmSetting/getTimeFormat/).
